cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsHandlerTestRun.java

```diff
@@ -94,7 +99,13 @@ private static void setupMetrics() throws Exception {
   public void testSearchContext() throws Exception {
     // empty context
     verifySearchResultContains("/v3/metrics/search?target=childContext",
-                               ImmutableList.<String>of("myspace", "yourspace"));
+                       ImmutableList.<String>of("myspace", "system", "yourspace"));
+
+    // verify childContext in system context
+    verifySearchResultContains("/v3/metrics/search?target=childContext&context=system",
+                       ImmutableList.<String>of("system.-"));
```

Comment:
It comes from [here](https://github.com/caskdata/cdap/blob/develop/cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricRecordsWrapper.java#L244) and [here](https://github.com/caskdata/cdap/blob/develop/cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricRecordsWrapper.java#L246)